run_pseudobulk_benchmark.py

```diff
@@ -44,27 +45,29 @@
     # preprocess_scrna(adata, keep_genes=1200)
 elif BENCHMARK_DATASET == "CTI":
     adata = sc.read("/home/owkin/project/cti/cti_adata.h5ad")
-    preprocess_scrna(adata,
+    adata, filtered_genes = preprocess_scrna(adata,
```

Comment:
same question about `filtered_genes`

Reply:
Please see my comment under another one of your comments